Dockerfile.multi-stage

```diff
@@ -20,12 +20,15 @@ FROM quay.io/prometheus/busybox@sha256:${BASE_DOCKER_SHA}
 LABEL maintainer="The Thanos Authors"
 
 COPY --from=builder /go/bin/thanos /bin/thanos
+COPY docker-enntrypoint.sh /docker-enntrypoint.sh
 
 RUN adduser \
     -D `#Dont assign a password` \
     -H `#Dont create home directory` \
     -u 1001 `#User id`\
     thanos && \
-    chown thanos /bin/thanos
+    chown thanos /bin/thanos && \
+    chown thanos /docker-enntrypoint.sh && \
+    chmod +x /docker-enntrypoint.sh
 USER 1001
-ENTRYPOINT [ "/bin/thanos" ]
+ENTRYPOINT [ "/docker-enntrypoint.sh" ]
```

Comment:
File is called `docker-entrypoint.sh` but you have a typo here.

Reply:
oh, thank you.